### PR TITLE
Cloudwatch back-end

### PR DESCRIPTION
**Pull request by zefhemel**
Booyah

### CODE REVIEW

File: pkg/config/kibana.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package config
```
Comment (zefhemel) [inline pkg/config/kibana.go]: This isn't new, just moved it to a separate file.

File: pkg/backend/cloudwatch/client.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/egnyte/ax/pkg/backend/common"
+)
+
+type CloudwatchClient struct {
+	client    *cloudwatchlogs.CloudWatchLogs
```
Comment (romanlevin) [inline pkg/backend/cloudwatch/client.go]: Personally I'd call this `logs`:

- Consistency with the type
- Avoid `client.client`

Comment (zefhemel) [reply pkg/backend/cloudwatch/client.go]: Makes sense

Comment (zefhemel) [reply pkg/backend/cloudwatch/client.go]: Done

**Comment by zefhemel** [comment]
@romanlevin added comments, please recheck and merge if OK

File: pkg/backend/common/client.go [hunk 3]
```diff
@@ -182,3 +183,65 @@ func MatchesQuery(m LogMessage, q Query) bool {
 	}
 	return matchFound
 }
+
+func ReQueryFollow(ctx context.Context, resultChan chan LogMessage, queryMessagesFunc func() ([]LogMessage, error)) {
```
[the remark this answers is not in the document]
Comment (zefhemel) [reply pkg/backend/common/client.go]: Done

**Pull request merged**